benches/bench.rs

```diff
@@ -9,10 +9,12 @@ use smallvec::{ExtendFromSlice, SmallVec};
 
 const VEC_SIZE: usize = 16;
 const SPILLED_SIZE: usize = 100;
+const LARGE_SIZE: usize = 10000;
 
 trait Vector<T>: for<'a> From<&'a [T]> + Extend<T> + ExtendFromSlice<T> {
     fn new() -> Self;
     fn push(&mut self, val: T);
+    fn push_light(&mut self, val: T);
```

Comment:
IMO this should either not exist when the feature is not enabled, or delegate to `push` in the default implementation.